challenges/2021-06-29-run-of-ones/src/main/java/io/codeconnector/codedojo/RunOfOnes.java

```diff
@@ -0,0 +1,17 @@
+package io.codeconnector.codedojo;
```

Comment:
The newer version of the Java challenge setup includes the code and tests in a single file/class.